woocommerce/woocommerce
# Product Gallery: Simplify the Product Gallery Pop-up

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
Since the designs and implementation details have not yet been decided upon (and the discussions are still open: pdnLyh-4M4-p2), with the deadline for launching the Product Gallery approaching, we have agreed to settle on an interim solution for now, which is a stripped-down, simplified version of the Product Gallery Pop-up consisting of a Large Image and Next/Previous buttons.

This PR:
- Updates the SVG Outside Next/Previous Button arrow icons.
- Hides the Product Gallery template part from FSE (as the current Product Gallery block used for the pop-up is not ready to be exposed in that context).
- Update the Product Gallery template part contents to match the new, simplified Product Gallery Pop-up design.
- Simplifies and fixes the Pop-up styles.

The new Pop-up design reference:
<img width="688" alt="Jarek_s_scratchpad_–_Figma" src="https://github.com/woocommerce/woocommerce/assets/905781/048f6820-10d0-43d6-96fc-7445a0e22702">





Closes #42044 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Appearance > Editor > Templates > Single Product (clear customizations)
2. Click on the editor and you should see "Transform into blocks" action button.
3. Add the Product Gallery block.
4. Make sure the Product Gallery `Full-screen when clicked` setting is enabled.
5. Open a single product page for a product that has multiple images added to the Product gallery. It's best to test with various images: square, horizontal, vertical, small, large etc.
6. Click on the Large Image (later on try again by clicking on the View All thumbnails overlay).
7. Ensure the Product Gallery Pop-up displays correctly for various viewport sizes.
8. Make sure you can navigate through all available images.
9. Make sure the Close button works correctly. 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
